src/main/kotlin/com/codingnomads/mentor_portal_api/entities/business/UserPostPayload.kt

```diff
@@ -0,0 +1,31 @@
+package com.codingnomads.mentor_portal_api.entities.business
+
+import com.codingnomads.mentor_portal_api.entities.data.UserRow
+import com.codingnomads.mentor_portal_api.entities.business.User
+
+data class UserPostPayload (
+    /**
+     * user fields
+     */
+    val id: Int,
+    val firstName: String,
+    val lastName: String,
+    val roleCode: Int,
+    val statusCode: Int,
+    val timezoneOffset: Int,
+    /**
+     * contact fields
+     */
+    val userId: Int,

```

Comment:
We will not have the userId at the time of mapping this payload, please remove this property.